packages/sit-onyx/src/styles/globals/root.scss

```diff
@@ -0,0 +1,30 @@
+@layer onyx.utility {
```

Comment:
Instead of putting all these general styles in a `utility` layer, can't we define a general `onyx` layer instead?

Reply:
See in [packages/sit-onyx/src/styles/globals/layers.scss](https://github.com/SchwarzIT/onyx/pull/768/files/20d7fb6c28eb4bdb8a515bd85c4f7a98ff6fe7c5#diff-7d8e57f01d19f241269bebb9d98720661ce8bdb62b45d21d46e28c6dff96cf28) we only define sub-layers.